Explorer/Assets/Scripts/Global/StaticContainer.cs

```diff
@@ -40,8 +40,8 @@ namespace Global
     public class StaticContainer : IDCLPlugin<StaticSettings>
     {
         private ProvidedInstance<CharacterObject> characterObject;
-        public readonly ObjectProxy<World> GlobalWorldProxy = new ();
-        public readonly ObjectProxy<AvatarBase> MainPlayerAvatarBaseProxy = new ();
+        public ObjectProxy<World> GlobalWorldProxy = new ();
+        public ObjectProxy<AvatarBase> MainPlayerAvatarBaseProxy = new ();
```

Comment:
why to remove `readonly`?

Reply:
It was a problem when merging main, will add them back

Reply:
Tackled in [this commit](https://github.com/decentraland/unity-explorer/pull/336/commits/2ff94b5bfe636a43c73c3a8616c9bca7b55d70cb)

Reply:
Good